appuio/k8up/templates/prometheus/prometheusrule.yaml

```
@@ -43,6 +37,15 @@ spec:
             severity: critical
           annotations:
             summary: "K8up jobs are stuck in {{ "{{ $labels.namespace }}" }} for the last 24 hours."
+        {{- range tuple "archive" "backup" "check" "prune" "restore" }}
```

I don't think we need alert rules for restores. @Kidswiss WDYT?
```suggestion
        {{- range tuple "archive" "backup" "check" "prune" }}
```

---

Why not though? So you don't have to actively check if it failed or not. Prometheus will tell you. I'd leave it in.

---

Alright.
Now that I think about it, it could be that people might want to disable those alerts for certain job types. Can this tuple made configurable @bastjan ?

---

Making it configurable seems the most reasonable solution here.